README.md

```diff
@@ -56,52 +56,44 @@ control it using keyboard and mouse.
 
 ### Summary
 
- - Linux: `apt install scrcpy`
+ - Linux (Debian and Ubuntu): `apt install scrcpy`
  - Windows: [download][direct-win64]
- - macOS: `brew install scrcpy`
+ - macOS ([Homebrew](https://brew.sh/)): `brew install scrcpy`  
+   \* You need `adb`, accessible from your `PATH` *\*.
 
-Build from sources: [BUILD] ([simplified process][BUILD_simple])
-
-[BUILD]: BUILD.md
-[BUILD_simple]: BUILD.md#simple
-
-
-### Linux
-
-On Debian and Ubuntu:
-
-```
-apt install scrcpy
+*\*
+```bash
+brew install android-platform-tools
 ```
 
-On Arch Linux:
+### More
+
+**Linux > Arch Linux**
 
 ```
 pacman -S scrcpy
 ```
 
-A [Snap] package is available: [`scrcpy`][snap-link].
+**Linux > [Snap] package**
 
-[snap-link]: https://snapstats.org/snaps/scrcpy
+[`scrcpy`](https://snapstats.org/snaps/scrcpy)
 
 [snap]: https://en.wikipedia.org/wiki/Snappy_(package_manager)
 
-For Fedora, a [COPR] package is available: [`scrcpy`][copr-link].
+**Linux > Fedora / [COPR] package**
+
+[`scrcpy`](https://copr.fedorainfracloud.org/coprs/zeno/scrcpy/)
 
 [COPR]: https://fedoraproject.org/wiki/Category:Copr
-[copr-link]: https://copr.fedorainfracloud.org/coprs/zeno/scrcpy/
 
+**Linux > Gentoo / [Ebuild]**
 
-For Gentoo, an [Ebuild] is available: [`scrcpy/`][ebuild-link].
+Available at [`scrcpy/`](https://github.com/maggu2810/maggu2810-overlay/tree/master/app-mobilephone/scrcpy).
 
 [Ebuild]: https://wiki.gentoo.org/wiki/Ebuild
-[ebuild-link]: https://github.com/maggu2810/maggu2810-overlay/tree/master/app-mobilephone/scrcpy
-
-You could also [build the app manually][BUILD] ([simplified
-process][BUILD_simple]).
 
 
-### Windows
+**Windows > archive**
```

Comment:
I think I prefer the current version, with 1 section for linux, 1 section for Windows and 1 section for macOS.